examples/dat3m/kit/dat3m.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (C) 2024 Vrije Universiteit Brussel. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+from pythainer.builders import (
+    DockerBuilder,
+    DockerfileDockerBuilder,
+    UbuntuDockerBuilder,
+)
+from pythainer.builders.utils import project_git_clone
+from pythainer.examples.builders import get_user_builder
+
+from benchkit.utils.git import clone_repo
+
+
+def vsyncer_builder() -> DockerBuilder:
```

Comment:
Why is vsyncer part of dat3m.py?

Reply:
I thought to support both ways to use dat3m. But maybe vsyncer deserves its own file/example directory.